A2rchi/interfaces/chat_app/app.py

```diff
@@ -105,63 +105,77 @@ def __call__(self, history: Optional[List[Tuple[str, str]]], discussion_id: Opti
         """
         Execute the chat functionality.
         """
-        self.lock.acquire()
-        print("INFO - acquired lock file")
         try:
             # update vector store through data manager; will only do something if new files have been added
+            self.lock.acquire()
```

Comment:
This self.lock.acquire should get moved out of the try, in front of it. This way, if something weird happens and we fail before/during execution of line 110, we won't silently fail.

Reply:
Good catch, just fixed